blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
```

Comment:
Ok; I was attempting to avoid a whole mess of separate state vars, since from a display perspective they're mutually exclusive and I'd assumed it would only apply to ui state, but I can see that it's not necessarily just ui state here. I'll separate those out. Thanks!